src/status_im/extensions/core.cljs

```diff
@@ -609,6 +629,16 @@
                  :arguments   {:chain-id    :number
                                :on-success? :event
                                :on-failure? :event}}
+                'show-fullscreen
```

Comment:
How about `screen/open` and `screen/close` ?

Reply:
yeah..